### PR TITLE
Update timeline component and documentation

### PR DESCRIPTION
**Pull request by whatterz**

### CODE REVIEW

File: src/components/moj-timeline/README.md [hunk 1]
```diff
@@ -3,7 +3,81 @@
 - [Guidance](https://mojdt-design-system.herokuapp.com/components/timeline)
 - [Preview](https://mojdt-frontend.herokuapp.com/components/timeline)
 
+## Dependencies
```
Comment (adamsilver) [inline src/components/moj-timeline/README.md]: I don't think we need to specify any of this here. The dependency should be included in the package

Comment (whatterz) [reply src/components/moj-timeline/README.md]: So in the `package/package.json` should include the following?
```
"dependencies": {
  "moment": "^2.22.2"
}
```

File: app/filters.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-const moment = require('moment');
 
 module.exports = function (env) {
```
Comment (adamsilver) [inline app/filters.js]: Don't need to pass in `env` I don't think?

File: app/filters.js [hunk 3]
```diff
@@ -9,58 +8,6 @@ module.exports = function (env) {
    */
   let filters = {}
 
-  /* ------------------------------------------------------------------
-    date filter for use in Nunjucks
-    example: {{ params.date | date("DD/MM/YYYY") }}
-    outputs: 01/01/1970
-  ------------------------------------------------------------------ */
-  filters.date = function(timestamp, format) {
```
Comment (adamsilver) [inline app/filters.js]: We shouldn't expose this. But I think you said you wanted to raise an issue and deal with this outside of this ticket?

File: src/components/moj-timeline/README.md [hunk 4]
```diff
@@ -3,7 +3,25 @@
...
+
+- [Moment.js](https://momentjs.com/)
+
+### Installation
```
Comment (adamsilver) [inline src/components/moj-timeline/README.md]: This is not specific to timeline, these filters are available at the package level and so should be provided in the main readme/installation guides if they exist. They will in the design system.

File: src/filters/all.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
+const moment = require('moment');
+
+module.exports = function (env) {
```
Comment (adamsilver) [inline src/filters/all.js]: Don't need `env` I don't think

File: app/filters.js (hunk 3 above)
Comment (whatterz) [reply app/filters.js]: Yes, a logging an issue is preferable. We need to consider what formats are used, for example "DD/MM/YYYY" is not encapsulated, yet used.

File: src/components/moj-timeline/README.md (hunk 4 above)
Comment (whatterz) [reply src/components/moj-timeline/README.md]: I have removed the line mentioning `moment.js` but will leave the installation instructions in lieu of another place to put the information.

File: app/filters.js (hunk 2 above)
Comment (whatterz) [reply app/filters.js]: This is the default construct for `app/filters.js`. I will leave as is and we can review at another time. I will create a ticket.

File: src/filters/all.js (hunk 5 above)
Comment (whatterz) [reply src/filters/all.js]: Removed.

**Pull request merged**